IdentityCore/src/webview/systemWebview/ios/MSIDAuthenticationSession.m

```diff
@@ -199,5 +198,23 @@ - (void)notifyEndWebAuthWithURL:(NSURL *)url
     }
 }
 
+- (BOOL)handleURLResponse:(NSURL *)url
+{
+    [[MSIDTelemetry sharedInstance] stopEvent:_telemetryRequestId event:_telemetryEvent];
+    
+    if (@available(iOS 12.0, *))
```

Comment:
Double check that this is not going to call the cancellation callback before we can execute with notifyEndWebAuthWithURL.

Reply:
checked, authentication session has no delegates in current context. 
thus this is fine